appveyor.yml

```diff
@@ -0,0 +1,85 @@
+    #   Can also be done via anaconda CLI with
+    #     $ anaconda auth --create --name my_appveyor_token
+    # 2. Generated secure env var below via appveyor's Encrypt data tool at https://ci.appveyor.com/tools/encrypt.
+    #   See https://www.appveyor.com/docs/build-configuration/#secure-variables.
+    ANACONDA_TOKEN:
+      secure: "ryxTeOgjDvz6Y25wr29GcUm1Y7iCOeNX96t1iXIp4bQzWCQszJ68+M6Zy6Kpg0C6"
+
+    CONDA_ROOT_PYTHON_VERSION: "2.7"
+
+    PYTHON_ARCH: "64"
+    PANDAS_VERSION: "0.18.1"
+    NUMPY_VERSION: "1.11.1"
+    SCIPY_VERSION: "0.17.1"
+
+  matrix:
+    - PYTHON_VERSION: "2.7"
+    - PYTHON_VERSION: "3.5"
+
+# We always use a 64-bit machine, but can build x86 distributions
+# with the PYTHON_ARCH variable (which is used by CMD_IN_ENV).
+platform:
+  - x64
+
+cache:
+  - '%LOCALAPPDATA%\pip\Cache'
+
+# all our python builds have to happen in tests_script...
+build: false
+
+init:
+  - "ECHO %PYTHON_VERSION% %PYTHON_ARCH% %PYTHON%"
+  - "ECHO %NUMPY_VERSION%"
+
+install:
+  # If there is a newer build queued for the same PR, cancel this one.
+  # The AppVeyor 'rollout builds' option is supposed to serve the same
+  # purpose but it is problematic because it tends to cancel builds pushed
+  # directly to master instead of just PR builds (or the converse).
+  # credits: JuliaLang developers.
+  - ps: if ($env:APPVEYOR_PULL_REQUEST_NUMBER -and $env:APPVEYOR_BUILD_NUMBER -ne ((Invoke-RestMethod `
+        https://ci.appveyor.com/api/projects/$env:APPVEYOR_ACCOUNT_NAME/$env:APPVEYOR_PROJECT_SLUG/history?recordsNumber=50).builds | `
+        Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
+          throw "There are newer queued builds for this pull request, failing early." }
+
+  - ps: $NPY_VERSION_ARR=$env:NUMPY_VERSION -split '.', 0, 'simplematch'
+  - ps: $env:CONDA_NPY=$NPY_VERSION_ARR[0..1] -join ""
+  - ps: $PY_VERSION_ARR=$env:PYTHON_VERSION -split '.', 0, 'simplematch'
+  - ps: $env:CONDA_PY=$PY_VERSION_ARR[0..1] -join ""
+  - SET PYTHON=C:\Python%CONDA_PY%_64
+  # Get cygwin's git out of our PATH.  See https://github.com/omnia-md/conda-dev-recipes/pull/16/files#diff-180360612c6b8c4ed830919bbb4dd459
+  - "del C:\\cygwin\\bin\\git.exe"
+  # this installs the appropriate Miniconda (Py2/Py3, 32/64 bit),
+  - powershell .\ci\appveyor\install.ps1
+  - SET PATH=%PYTHON%;%PYTHON%\Scripts;%PATH%
+
+  - conda info -a
+  - conda install conda=4.3.30 conda-build=3.0.28 anaconda-client=1.6.3 --yes -q
+  - conda list
+  # https://blog.ionelmc.ro/2014/12/21/compiling-python-extensions-on-windows/ for 64bit C compilation
+  - ps: copy .\ci\appveyor\vcvars64.bat "C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\bin\amd64"
+
+  - conda create -n testenv --yes -q --use-local pip python=%PYTHON_VERSION% numpy=%NUMPY_VERSION% scipy=%SCIPY_VERSION% -c quantopian -c https://conda.anaconda.org/quantopian/label/ci
```

Comment:
`--use-local` Are we building other packages locally that this package depends on? Or is this future-proofing?

`-c https://conda.anaconda.org/quantopian/label/ci` Same question - are there ci packages on anaconda that we expect to install here, or is this future-proofing?

Reply:
removed both of those - had thought of it as future proofing, but I don't think we need it as of now